### ISSUE TITLE
Live cd doesn't boot without acpi=off on chromebook

### ISSUE THREAD

**Issue by danielstuart14** [clearlinux/distribution#1098]
When trying to boot the latest desktop live cd (30670) on my Acer R11 (Braswell), system hangs on boot. 
I can only boot if acpi=off is added to the cmdline, but then trackpad and keyboard don't work. 

Any other distro boots flawlessly (fedora 30 & rawhide, ubuntu 18.04 & 19.04, etc).
My guess is that clear kernel is missing some chrome-specific kernel configs.

Fedora 30 kernel config enables:
CONFIG_CHARGER_CROS_USBPD=m
CONFIG_CHROME_PLATFORMS=y
CONFIG_CHROMEOS_LAPTOP=m
CONFIG_CHROMEOS_PSTORE=m
CONFIG_CROS_EC_I2C=m
CONFIG_CROS_EC_PROTO=y
CONFIG_CROS_EC_SPI=m

**Comment by lebensterben** [comment]
>When trying to boot the latest desktop live cd (30670) on my Acer R11 (Braswell), system hangs on boot.  ...

@danielstuart14 When boot up, press and hold space until the boot menu of sysmted-boot appears, press e and append those kernel parameters. Then press enter to see whether it works.

BTW, I also see the acpi error but it doesn't stop me from booting up.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by danielstuart14** [comment]
@lebensterben these aren't kernel parameters, but kernel configs that can only be set when compiling the kernel.

**Comment by amygilliam1** [comment]
@miguelinux , do we have these configs set in the native kernel?

**Comment by lebensterben** [comment]
> @lebensterben these aren't kernel parameters, but rather kernel configs that can only be set when compiling the kernel.

Yes you're right. There're more chromebook related kernel configurations [here](https://cateee.net/lkddb/web-lkddb/index_C.html), which contains "*_CHROMEOS_*, "*_CHROME_*", or "*_CROS_*".

**Comment by danielstuart14** [comment]
@lebensterben yes, these are related to chrome-ec sensors, nice to have, but can be added later. (Afaik, fedora doesn't even ship these enabled, ubuntu does)

**Comment by miguelinux** [comment]
we enabled Chrome hardware at **linux-5.2.9-825**

**Comment by danielstuart14** [comment]
Still not booting without acpi=off, maybe a coreboot issue?

**Comment by miguelinux** [comment]
@danielstuart14 what kernel version is using your CL installer?

**Comment by miguelinux** [comment]
@danielstuart14 can you post the output of `lsmod` on fedora or ubuntu when booting in your platform.

**Comment by danielstuart14** [comment]
> @danielstuart14 what kernel version is using your CL installer?

I'll check once I get home, but I tested the latest iso build from that day.

Here is the lsmod output on ubuntu 18.04: https://pastebin.com/xXKGeBHs

**Comment by danielstuart14** [comment]
Just checked, kernel version 5.2.10-828
